plutus-tx/src/PlutusTx/Lift/TH.hs

```diff
@@ -64,29 +65,30 @@ import Prelude as Haskell
 type RTCompileScope uni fun = ReaderT (LocalVars uni) (RTCompile uni fun)
 type THCompile = StateT Deps (ReaderT THLocalVars (ExceptT LiftError TH.Q))
 
-data LiftError = UnsupportedLiftKind !TH.Kind
-               | UnsupportedLiftType !TH.Type
-               | UserLiftError !T.Text
-               | LiftMissingDataCons !TH.Name
-               | LiftMissingVar !TH.Name
-               deriving anyclass (Prelude.Exception)
+data LiftError
+  = UnsupportedLiftKind !TH.Kind
+  | UnsupportedLiftType !TH.Type
+  | UserLiftError !T.Text
+  | LiftMissingDataCons !TH.Name
+  | LiftMissingVar !TH.Name
+  deriving anyclass (Prelude.Exception)
 
 instance PP.Pretty LiftError where
-    pretty (UnsupportedLiftType t) = "Unsupported lift type: " PP.<+> PP.viaShow t
-    pretty (UnsupportedLiftKind t) = "Unsupported lift kind: " PP.<+> PP.viaShow t
-    pretty (UserLiftError t)       = PP.pretty t
-    pretty (LiftMissingDataCons n) = "Constructors not created for type: " PP.<+> PP.viaShow n
-    pretty (LiftMissingVar n)      = "Unknown local variable: " PP.<+> PP.viaShow n
+  pretty (UnsupportedLiftType t) = "Unsupported lift type: " PP.<+> PP.viaShow t
```

Comment:
Yeah, being inconsistent about alignment is one of the annoyances with our style. I think ormolu/fourmolu tends to avoid alignment in the interest of diff-friendliness

Reply:
> I think ormolu/fourmolu tends to avoid alignment in the interest of diff-friendliness

I think we had a discussion like this before and I accused you of putting the convenience of machines before the convenience of human beings.  However, I suppose that if I really want to align thing to look at them I can get emacs to do it temporarily and let the formatter change it back if it wants to.